lib/core/viewer/components/viewer.component.ts

```diff
@@ -32,16 +32,25 @@ import { Subscription } from 'rxjs';
 import { ViewUtilService } from '../services/view-util.service';
 import { AppExtensionService, ViewerExtensionRef } from '@alfresco/adf-extensions';
 import { filter } from 'rxjs/operators';
+import { VersionEntry } from '@alfresco/js-api/src/api/content-rest-api/model/versionEntry';
+import { Version } from '@alfresco/js-api/src/api/content-rest-api/model/version';
 
 @Component({
     selector: 'adf-viewer',
     templateUrl: './viewer.component.html',
     styleUrls: ['./viewer.component.scss'],
-    host: { 'class': 'adf-viewer' },
+    host: {'class': 'adf-viewer'},
     encapsulation: ViewEncapsulation.None
 })
 export class ViewerComponent implements OnChanges, OnInit, OnDestroy {
 
+    constructor(private apiService: AlfrescoApiService,
```

Comment:
move back the contractor where it was has to be after the properties

Reply:
auto align did it, didnt even realised lol